resource/ui/ts/models/proto.ts

```diff
@@ -37,6 +37,48 @@ module Models {
             last_update_nanos: number;
         }
 
+        /**
+         * Create a new object which implements MVCCStats interface, with zero
+         * values.
+         */
+        export function NewMVCCStats():MVCCStats {
```

Comment:
Perhaps we should move the proto interfaces to a different file from the implementations.  
Put the interfaces into /typeings/proto.ts.
And any implementation into their own model files.  If everything is in a single large file, than it will get too busy.